idea_town/frontend/static-src/test/general.js

```diff
@@ -0,0 +1,21 @@
+var test = require('tape');
+var path = require('path');
+var fs = require('fs');
+
+test(`🖳  Running Tests for ${__filename} ␄`, function(a) {
```

Comment:
I'm removing them for this test. Since this test is run outside of the babel transform.